vignettes/advanced_classification_workflows.Rmd

```diff
@@ -36,7 +36,7 @@ library(ComplexHeatmap)
 
 # Introduction
 
-This vignette demonstrates advanced usage patterns for the main classifier in LundTaxR, building upon the concepts introduced in the "Getting Started" guide. If you're new to LundTaxR, we recommend starting with the basic tutorial first:
+This vignette demonstrates advanced functions/parameters? for the main classifier(The only one?) in LundTaxR, building upon the concepts introduced in the "Getting Started" guide. If you're new to LundTaxR, we recommend starting with the basic tutorial first:
```

Comment:
Perhaps we can say `advanced parameter usage`? For your second comment, yes, we should change this. Perhaps the full sentence could be; 

```
This vignette demonstrates advanced parameter usage for the LundTax classifier, building upon the concepts introduced in the "Getting Started" guide.
```